CONTRIBUTING.md

```diff
@@ -98,6 +98,17 @@ java:path/to/java/file.java@Region
 kotlin:path/to/kotlin/file.kt@Region
 ```
 ~~~
+If you write java code, the wiki has a tool to generate translations from quilt mappings to mojmaps; simply run the
+`translateToMojmaps` task to generate translated code of any code in `src/main/java` of every project. To include these
+translations in the displayed code on a page, use the following syntax:
+~~~markdown
+```tabbed-files
+java:src/main/java/package/File.java@Region
```

Comment:
Does this default to qm? That should be stated since the examples include `@qm`.

Reply:
It only defaults to QM if at least one provided entry has mappings; otherwise, it has no mappings